src/views/explore/explore.jsx

```diff
@@ -0,0 +1,159 @@
+        if (this.props.tab != 'all') {
+            qText = '&q=' + this.props.category;
+        }
+        this.api({
+            uri: '/search/' + this.props.itemType
+                + '?limit=' + this.props.loadNumber + '&offset=' + this.state.offset + qText
+        }, function (err, body) {
+            if (!err) {
+                var loadedSoFar = this.state.loaded;
+                Array.prototype.push.apply(loadedSoFar,body);
+                this.setState({loaded: loadedSoFar});
+                var currentOffset = this.state.offset + this.props.loadNumber;
+                this.setState({offset: currentOffset});
+            }
+        }.bind(this));
+    },
+    changeItemType: function () {
+        var newType;
+        for (var t in this.props.acceptableTypes) {
+            if (this.props.itemType != t) {
+                newType = t;
+                break;
+            }
+        }
+        window.location = window.location.origin + '/explore/'+newType+'/'+this.props.tab;
+    },
+    getBubble: function (type) {
+        var allBubble = <a href={'/explore/'+this.props.itemType+'/'+type+'/'}>
+                        <li>
+                            <FormattedMessage
+                                id={'explore.' + type}
+                                defaultMessage={type.charAt(0).toUpperCase() + type.slice(1)} />
+                        </li>
+                    </a>;
+        if (this.props.category==type) {
+            allBubble = <a href={'/explore/'+this.props.itemType+'/' + type + '/'}>
+                        <li className='active'>
+                            <FormattedMessage
+                                id={'explore.' + type}
+                                defaultMessage={type.charAt(0).toUpperCase() + type.slice(1)} />
+                        </li>
+                    </a>;
+        }
+        return allBubble;
+    },
+    getTab: function (type) {
+        var allTab = <a href={'/explore/'+type+'/'+this.props.category+'/'}>
+                        <li>
+                            <FormattedMessage
+                                id={'explore.' + type}
+                                defaultMessage={type.charAt(0).toUpperCase() + type.slice(1)} />
+                        </li>
+                    </a>;
+        if (this.props.itemType==type) {
+            allTab = <a href={'/explore/'+type+'/' + this.props.category + '/'}>
+                        <li className='active'>
+                            <FormattedMessage
+                                id={'explore.' + type}
+                                defaultMessage={type.charAt(0).toUpperCase() + type.slice(1)} />
+                        </li>
+                    </a>;
+        }
+        return allTab;
+    },
+    render: function () {
+        return (
+            <div>
+                <div className='outer'>
+                    <Box title={'Explore'}>
+                        <SubNavigation className='categories'>
```

Comment:
Are we getting rid of the custom tag search in this subnav in favor of the new search page?